src/utils/index.ts

```diff
@@ -146,3 +146,15 @@ export function resolveAliases (_aliases: Record<string, string>) {
   }
   return aliases
 }
+
+export async function retry (fn: () => Promise<void>, retries: number) {
+  let retry = 0
+  let error: any
+  while (retry++ < retries) {
+    try { return await fn() } catch (err) {
+      error = err
```

Comment:
We might always show error as warning. If it is happening too often not hiding the issue.